tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -882,14 +882,18 @@ def _select_case(self, role, case):
         finder = re.compile(r'.*\b' + case + r'\b')
         finder_dotted = re.compile(r'.*\b' + case.replace(' ', r'\.') + r'\b')
         for elem in elems:
-            action_chains = ActionChains(self._browser)
-            action_chains.move_to_element(elem)
-            action_chains.perform()
-            logger.debug(elem.text)
-            if finder.match(elem.text) or finder_dotted.match(elem.text):
-                parent = elem.find_element_by_xpath('..')
-                checkbox = parent.find_element_by_class_name('tree-checkbox')
-                break
+            if elem.text:
+                self._browser.execute_script('arguments[0].scrollIntoView();',
```

Comment:
@FiveDimensions would  you please add comments to illustrate when `elem.text` is null and what `arguments[0]` is?

Also noticed your comment of issue #4530:

> This issue happens on the testbeds with mixed vendor devices using Chrome 80 and Chromedriver 80. The ActionChains.perform() will throw an exception when ActionChains move to the element without text. 

Would you please help to explain why it only happens on mixed testbed? what's the difference compared with CV testbed?

If possible, why it occurs when using the Chrome 80?

Reply:
@librasungirl "arguments[0].scrollIntoView();" is a javascript and will take the reference of the "elem" as arguments[0] to scroll the window to it. 
There are 2 issues using Chrome and chromedriver 80:
1. Cases will not display when the selected vendor devices are not enough, their web classes are tagged as "NotToBeEnabled" and their web elements' text is null. When elem.text is null, move_to_element(elem) and perform() will throw exceptions. 
2. move_to_element(elem) will not work when the "elem" class is not displayed in the current window and need to scroll to it.
 For case Router 5.1.7,  it needs 11 devices,  in CV testbed, all previous cases (Router 5.1.1, Router 5.1.2 ....) in the list can display since their needed device numbers are less than 11 which means move_to_element(elem) and perform() work. In IV (mixed vendor) testbed, several previous cases do not show in the list since the vendor devices are not enough, which will cause the issue #1.

Reply:
I read @librasungirl's comment as adding a comment/documentation to the code - @librasungirl, can you confirm?

Reply:
yes, add comments into code for readability.

Reply:
Thanks for the explanation. Still I have one question:
> Cases will not display when the selected vendor devices are not enough, their web classes are tagged as "NotToBeEnabled" and their web elements' text is null. 

For CV testbed, there are also chances that the reference devices may be not enough, right? it should be a common issue for both IV/CV testbeds?

Reply:
Yes, it should be